openedx/features/course_experience/utils.py

```diff
@@ -270,8 +270,11 @@ def reset_deadlines_banner_should_display(course_key, request):
                 if display_reset_dates_banner:
                     break
                 for subsection in section.get('children', []):
-                    if (not subsection.get('complete', True)
-                            and subsection.get('due', timezone.now() + timedelta(1)) < timezone.now()):
+                    if (
+                        not subsection.get('complete', True)
+                        and subsection.get('graded', False)
```

Comment:
And scored too?

Reply:
No. The subsection is graded. The content inside might be scored. But the subsection being marked graded means that an instructor selected to grade it as an assignment.